Helm chart missing required apiVersion

**Describe the bug**:
`Chart.yaml` is missing the required `apiVersion`.

**Expected behaviour**:
`helm lint` exits successfully.

**Steps to reproduce the bug**:
```bash
helm repo add jetstack https://charts.jetstack.io
helm repo update jetstack
helm fetch jetstack/cert-manager --version v0.11.0                                                                                     
helm lint ./cert-manager-v0.11.0.tgz
```
```
==> Linting ./cert-manager-v0.11.0.tgz
[ERROR] Chart.yaml: apiVersion is required

Error: 1 chart(s) linted, 1 chart(s) failed
```

**Anything else we need to know?**:
`apiVersion` is required, as per [the Helm docs](https://helm.sh/docs/developing_charts/#the-chart-yaml-file). It is enforced in v2.15.1.

**Environment details:**:
- Kubernetes version: v1.14.6
- Cloud-provider/provisioner: n/a (AWS)
- cert-manager version: v0.11.0
- Install method: Helm v2.15.1

/kind bug

--- a/deploy/charts/cert-manager/Chart.yaml
+++ b/deploy/charts/cert-manager/Chart.yaml
@@ -1,3 +1,4 @@
+apiVersion: v1
 name: cert-manager
 # The version and appVersion fields are set automatically by the release tool
 version: v0.1.0

